web/src/components/AppRouter.tsx

```diff
@@ -0,0 +1,16 @@
+import { Router } from '@reach/router';
```

Comment:
@yhfreeman Any new files need the to have the licensing header at the top; i've forgotten it in a few places but we should fix it going fwd. Check out the header at the top of constants.ts as an example.